resources/assets/js/components/Dashboard/Assignments.js

```diff
@@ -0,0 +1,74 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+import ReactDOM from 'react-dom';
+
+export default class Assignments extends Component {
+  constructor() {
+    super();
+    this.state = {
+      tasks: [],
+      progress_bar_width: 40
```

Comment:
Default to 0? So it doesn't look like the progress bar jumps from 40 to another value